app/logbook/elog/src/main/java/org/phoebus/elog/api/ElogApi.java

```diff
@@ -385,9 +382,7 @@ public File getAttachment( String filename ) throws LogbookException {
           types.add( t );
         }
       }
-    } catch (XPathExpressionException e) {
-      throw new LogbookException( e.getMessage() );
-    } catch (ParserConfigurationException e) {
+    } catch (XPathExpressionException | ParserConfigurationException e) {
       throw new LogbookException( e.getMessage() );
```

Comment:
Instead of `throw new OtherException(original.getMessage())`, I would suggest `throw new OtherException("Stuff happened", original)`.
For one, `original.getMessage()` may be `null`, so you get nothing useful.
Additionally, passing the 'cause' up in the wrapped message allows for "Caused by: ..." and the full stack trace to appear in the, well, stack trace.

Reply:
You are right. This needs to be adapted.

Reply:
`hrow new LogbookException( "could not parse the elog response", e )` is good, thanks!